res/layout/list_fragment.xml

```diff
@@ -40,19 +48,14 @@
         android:layout_height="match_parent"
         android:visibility="gone" >
 
-        <ScrollView
```

Comment:
Is it needed to remove this Scrollview? 
It was used to allow user to do swipe-refresh on layout when folder is empty...